packages/Owl/Owl.0.1.0/opam

```diff
@@ -0,0 +1,32 @@
+opam-version: "1.2"
+maintainer: "Liang Wang (ryanrhymes@gmail.com)"
+authors: [ "Liang Wang (ryanrhymes@gmail.com)" ]
+license: "MIT"
+homepage: "https://github.com/ryanrhymes/owl"
+dev-repo: "https://github.com/ryanrhymes/owl.git"
+bug-reports: "https://github.com/ryanrhymes/owl/issues"
+build: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--prefix" prefix]
+  ["ocaml" "setup.ml" "-build"]
+]
+install: ["ocaml" "setup.ml" "-install"]
+remove: [
+  ["ocaml" "%{etc}%/Owl/_oasis_remove_.ml" "%{etc}%/Owl"]
+]
+build-test: [
+  ["oasis" "setup"]
+  ["ocaml" "setup.ml" "-configure" "--enable-tests"]
+  ["ocaml" "setup.ml" "-build"]
+  ["ocaml" "setup.ml" "-test"]
+]
+build-doc: [ "ocaml" "setup.ml" "-doc" ]
+depends: [
+  "ctypes"
+  "dolog"
+  "gsl"
+  "oasis" {build & >= "0.4"}
+  "ocamlbuild" {build}
+  "ocamlfind" {build}
+  "plplot"
```

Comment:
Yep, sparse matrix support is very limited (only real numbers) in GSL and comes much later. I used brew to upgrade to GSL 2.2.1 :)

For complex sparse matrix, I make a pure OCaml implementation myself. I probably will do the same for sparse real matrix in the future so that the old GSL can work (I guess this is the only solution for this).